### PR TITLE
Automation Scripts - Settings Feature

### PR DESCRIPTION
**Pull request by PayalChoksey17**
Adding all the scripts for Settings Feature

### CODE REVIEW

File: app/lib/accounts.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const userEmailSchema = require("./userEmailSchema");
 AccountsTemplates.configure({
   // Behavior
-  forbidClientAccountCreation: true,
+  forbidClientAccountCreation: false,
```
Comment (brylie) [inline app/lib/accounts.js]: Please do not commit this change. When committing changes, only stage the files you want to commit. Avoid staging all files with `git add .`, as you may commit unintended files.

**Comment by brylie** [comment]
The project name is **GeriLife**. Please use the correct project name in your tests.

File: app/lib/accounts.js (hunk 1 above)
Comment (PayalChoksey17) [reply app/lib/accounts.js]: Alright. Noted!

**Comment by PayalChoksey17** [comment]
@brylie Can you point out where exactly I haven't used the Project name as " GeriLife" as I have already names my project as " GeriLife"

Project

**Comment by brylie** [comment]
> Can you point out where exactly I haven't used the Project name as " GeriLife"

Screenshot_20190626_101003

Double check the spelling. The name "**Giri**life" is used throughout the test files and even the folder, instead of **Geri**Life".

**Comment by PayalChoksey17** [comment]
@brylie 

oh oh!! bad mistake. Rectifying immediately. Thanks.

File: app/lib/accounts.js (hunk 1 above)
Comment (brylie) [reply app/lib/accounts.js]: Cool. It is still in the commit history, so please fix before I can merge this branch.

**Comment by PayalChoksey17** [comment]
@brylie  have updated accounts.js file.

File: tests/GeriLife/GeriLife.prj [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Project>
+   <description></description>
+   <name>GiriLife</name>
```
Comment (brylie) [inline tests/GeriLife/GeriLife.prj]: This is still using the wrong name "GiriLife".

**Comment by PayalChoksey17** [comment]
@brylie done with rectification.

**Pull request closed without merging**